nixos/modules/services/web-apps/matomo.nix

```diff
@@ -164,6 +164,7 @@ in {
             # Copy config folder
             chmod g+s "${dataDir}"
             cp -r "${cfg.package}/share/config" "${dataDir}/"
+            mkdir -p "${dataDir}/misc"
```

Comment:
Is this line needed ?

Reply:
Yes, we point `path.geoip2` to this directory -- I don't think Matomo calls `mkdir` by itself.